internal/checker/utilities.go

```diff
@@ -1219,7 +1219,68 @@ func isValidNumberString(s string, roundTripOnly bool) bool {
 }
 
 func isValidBigIntString(s string, roundTripOnly bool) bool {
-	return false // !!!
+	if s == "" {
```

Comment:
This implementation doesn't match the old compiler at all; rather than using a handwritten parser, we should just be using the scanner, like we used to do. See `isValidBigIntString` back in `src/compiler/utilities.ts`.

Reply:
updates in place for review